### DISCUSSION TITLE
Idea for a bigger screen

### DISCUSSION THREAD

**Discussion by Wapata** [alf45tar/PedalinoMini#450] [Ideas]
I there.
I was reading a lot of pages into this GitHub project and understood that some people want bigger screen on one side and there no more gpio on the other side.
... But we can think of the box ?
What about a new independent board ? With bigger screen or/and 3 big digit numbers ?
And what about that : there is no need for an API, the two boards can share a wifi link, or a midi one, can use OSC.
Tadaaaa
It's just an idea. But I didn't read something like this one until there, and tough it was worth to share it.

Have a nice day !!

**Comment by alf45tar** [comment]
Thanks for idea

**Comment by jumperrg** [comment]
Nice idea, do you mean this screen?  https://www.openhasp.com/0.6.1/devices/wt32-sc01/
